packages/view-types/link-controller/src/LinkControllerSubscriber.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import {
+  TitleInfo,
+} from '@vitessce/vit-s';
+import LinkController from './LinkController.js';
+
+export function LinkControllerSubscriber(props) {
+  const {
+    removeGridComponent,
+    theme,
+    title = 'Vitessce Link',
+    closeButtonVisible,
+    // Props for the link controller:
+    linkID,
+    authToken = 'mr-vitessce',
```

Comment:
```suggestion
    authToken = 'mr-vitessce',
```
I have moved these variables related to the link server infrastructure to here so they can be provided via props. However we should still consider how to best store/configure this info